components/deleteResourceLink/index.jsx

```diff
@@ -39,18 +44,33 @@ export default function DeleteResourceLink({
   onDelete,
   ...linkProps
 }) {
-  const { data: resourceInfo, error: resourceInfoError } = useResourceInfo(
-    resourceIri
-  );
-  const { accessControl, error: accessControlError } = useAccessControl(
-    resourceInfo
-  );
+  const { fetch } = useSession();
 
-  const handleDelete = createDeleteHandler(accessControl, onDelete);
+  const { policiesContainer } = usePoliciesContainer();
+  const [policyUrl, setPolicyUrl] = useState(null);
 
-  if (resourceInfoError || accessControlError) {
-    return null;
-  }
+  useEffect(() => {
+    if (!policiesContainer || !resourceIri) return;
+    (async () => {
+      const { response: resource, error } = await getResource(
+        resourceIri,
+        fetch
+      );
+      if (error) {
+        throw error;
```

Comment:
I think we should render an error message instead of throwing an error here. Remember that thrown errors can result in a blank page unless their picked up properly.